add-on/chrome/bootstrap.js

```diff
@@ -75,6 +75,25 @@ var WindowListener = {
       },
 
       /**
+       * @return {Promise} Getter for the Loop constants
+       */
+      get constants() {
+        return new Promise(resolve => {
```

Comment:
There's no need to wrap this in a Promise, because we _know_ that 'GetAllConstants' is synchronous.